ext/openssl/tests/openssl_pkey_get_public_rfc7512.phpt

```diff
@@ -0,0 +1,139 @@
+--TEST--
+openssl_pkey_get_public(), openss_pkey_get_private(), openssl_x509_read() with RFC7512 URI
+--SKIPIF--
+<?php
+if (!extension_loaded("openssl"))
+    die("skip");
+if (!function_exists("proc_open")) die("skip no proc_open");
+$PKCS11_MODULE_PATH="/usr/lib/softhsm/libsofthsm2.so";
+exec('openssl help', $out, $code);
+if ($code > 0) die("skip couldn't locate openssl binary");
+exec('softhsm2-util --version', $out, $code);
+if ($code > 0) die("skip couldn't locate softhsm2-util binary");
+exec('pkcs11-tool --show-info --module ' . $PKCS11_MODULE_PATH, $out, $code);
+if ($code > 0) die("skip couldn't locate pkcs11-tool binary");
+exec('pkcs11-dump info ' . $PKCS11_MODULE_PATH, $out, $code);
+if ($code > 0) die("skip couldn't locate pkcs11-dump binary");
+?>
+--FILE--
+<?php
+
+/* simple exec */
+function sexec($cmd, &$stdout=null, &$stderr=null) {
+  $proc = proc_open($cmd,[
```

Comment:
test files in openssl ext should have 4 spaces indent and wrapping of arguments sort of follows PSR 2 or 12 so something like this instead pls:
```
$proc = proc_open(
    $cmd,
    [
        1 => ['pipe','w'],
        2 => ['pipe','w'],
    ],
    $pipes
);
```

Reply:
done https://github.com/php/php-src/pull/6860/commits/d166313308ed1578647d55e542b1888694eaef9d